src/video-player.js

```diff
@@ -306,16 +306,16 @@ class VideoPlayer extends Utils.mixin(Eventable) {
 
         const publicId = source.publicId();
 
-        const transformations = source.transformation();
+        const transformation = Utils.assign({}, source.transformation());
```

Comment:
This fixes the compatibility issue

Reply:
what is the compatibility issue you are referring to here?

Reply:
The original issue that re-opened this - when seekThumbs is enabled it breaks playlist "coming-up" component & recommendations overlay.